AnnoyancesFilter/Widgets/sections/widgets.txt

```diff
@@ -169,6 +169,9 @@ blog.livedoor.jp##div[id^="shitarss_title_"]
 !
 ! NOTE: Blogroll specific rules
 !
+sensualgirls.org##.cbox2_cont.center
```

Comment:
```
sensualgirls.org##.cbox2_cont.center
```
it does not hide leftovers. Try to use `:has()` with matching `script` tag. But don't block the main content.